qiskit_ibm_runtime/options/execution_options.py

```diff
@@ -29,14 +30,34 @@ class ExecutionOptions:
     """Execution options.
 
     Args:
-        shots: Number of repetitions of each circuit, for sampling. Default: 4000.
+        shots: Number of repetitions of each circuit, for sampling. Default: 4096.
 
         init_qubits: Whether to reset the qubits to the ground state for each shot.
             Default: ``True``.
+            
+        samples: The number of samples of each measurement circuit to run. This
+            is used when twirling or resilience levels 1, 2, 3. If None it will
+            be calculated automatically based on the ``shots`` and
+            ``shots_per_sample`` (if specified).
+            Default: None
+        
+        shots_per_sample: The number of shots per sample of each measurement
+            circuit to run. This is used when twirling or resilience levels 1, 2, 3.
+            If None it will be calculated automatically based on the ``shots`` and
+            ``samples`` (if specified).
+            Default: None
+            
+        interleave_samples: If True interleave samples from different measurement
+            circuits when running. If False run all samples from each measurement
+            circuit in order.
+            Default: False
```

Comment:
Shouldn't these be resilience options? Given they are only applicable with error mitigation

Reply:
Its debatable. They were put here because they are intended to be able to be passed in as run options to `estimator.run`. I think interleave_samples can be moved to twirling options, but I think the other two should probably stay here assuming this catalogues things that can be passed in to `run`

Reply:
Actually today you can specify any of the options in `run`. The `execution` is just the group of options related to execution time settings applied to the input circuits, not related to `run`. Now one can argue `run` should only accept execution options, but unlike `backend.run`, `estimator.run` is a call to Estimator, hence I think it should take all options.

Reply:
For now I think it is best to keep them like this since this is what the internal code expects right now, this will avoid having to coordinate releases of all the dependencies to change this around. But I am open to moving them around later once all the pieces are also updated